### PR TITLE
Create REAEME.md

### PR DESCRIPTION
**Pull request by aiclaudev**
Co-Authored-By: SEONG PIL JU <69073013+JuSeongPil@users.noreply.github.com>

## What does this PR do? Please write it down in detail.
* What was the previous problem ?
There is no README.md in euphony-io/soundless-check-in.

* How did you solve it?
Through discussing, Add **Reason for making this application**, **Run the app**, **Report Issues**, **Library** category.

## Link to the related issue
[Issue #2](https://github.com/euphony-io/soundless-check-in/issues/2) 

## Is there anything you have tried to improve but failed?
I wanted to add 'What is next' category, but  i don't know what to write in this category.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Soundless-check-in
+
+![euphony_logo](https://user-images.githubusercontent.com/8490058/129358383-14b6f34d-9264-4c49-9538-3ef2e62a6490.png)
+
+This is a check-in application that identifies visitors, especially effective during the COVID-19.
+This application stores below informations in database
```
Comment (jopopcorn) [inline README.md:6]: I'd like to apply the enter one more time.
```suggestion
This is a check-in application that identifies visitors, especially effective during the COVID-19.

This application stores below informations in database.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## Report Issues
+
+To report a specific problem or feature request, [please open a new issue](https://github.com/euphony-io/soundless-check-in/issues/new/choose). 
+Your participation drives the development of the application!
+
+## Library
+
+It is based on Euphony library. 
+Official Facebook Page : https://www.facebook.com/euphonyproject
+Official Library Site : https://dev.jbear.co/euphony
```
Comment (jopopcorn) [inline README.md:44]: Please apply Enter to this part as well.
```suggestion
It is based on Euphony library. 

Official Facebook Page : https://www.facebook.com/euphonyproject

Official Library Site : https://dev.jbear.co/euphony
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+* ### Company
+
+<img src="https://user-images.githubusercontent.com/88221233/131173092-734265c0-2cfa-41a6-91c5-c2dcc38f916d.png"  width="250" height="400"/>  <img src="https://user-images.githubusercontent.com/88221233/131173182-922e3e0d-bfe4-45fb-a444-eea690742993.png"  width="250" height="400"/> <img src="https://user-images.githubusercontent.com/88221233/131173224-6939d699-1c70-4516-87b6-0bb442cb5f5e.png"  width="250" height="400"/>
+1. Press setting icon
+2. Enter the company information.
+
+## Report Issues
+
+To report a specific problem or feature request, [please open a new issue](https://github.com/euphony-io/soundless-check-in/issues/new/choose). 
+Your participation drives the development of the application!
```
Comment (jopopcorn) [inline README.md:38]: Please apply Enter to this part as well!
```suggestion
To report a specific problem or feature request, [please open a new issue](https://github.com/euphony-io/soundless-check-in/issues/new/choose). 

Your participation drives the development of the application!
```

**Review by jopopcorn — APPROVED** [review]
You've done a great job writing README!

**Comment by aiclaudev** [comment]
@jopopcorn 
Thank you! 
I updated file reflecting your feedback :)

**Comment by jopopcorn** [comment]
Great! Thanks your efforts 😄

**Review by sujin9 — APPROVED** [review]
Perfect! :)
And in 'What's next', i'd like to add phone verification service when user enter their phone number!

**Comment by aiclaudev** [comment]
> Perfect! :)
> And in 'What's next', i'd like to add phone verification service when user enter their phone number!

It's so great idea !!
After the application is completed, I will add it when I organize it for the last time.
Thanks for your feedback :)

**Comment by jopopcorn** [comment]
I would appreciate it if you could add a license when modifying README!

</br>

## License
soundless-check-in is licensed under the Apache 2.0 license. See the [LICENSE](https://github.com/euphony-io/soundless-check-in/blob/master/LICENSE) file for details.

**Pull request closed without merging**